### PR TITLE
Bump controller-runtime from 0.18.5 to 0.19.0

### PR DESCRIPTION
**Pull request by cheina97**
# Description

Bumps [sigs.k8s.io/controller-runtime](https://github.com/kubernetes-sigs/controller-runtime) from 0.18.5 to 0.19.0.

### CODE REVIEW

**Comment by cheina97** [comment]
This PR needs this PR to be merged https://github.com/virtual-kubelet/virtual-kubelet/pull/1247

File: pkg/liqo-controller-manager/networking/external-network/remapping/ip_controller.go [hunk 1]
```diff
@@ -86,7 +91,7 @@ func (r *IPReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Re
 		return ctrl.Result{}, nil
 	}
 
-	if ip.Status.IPMappings == nil || len(ip.Status.IPMappings) == 0 {
+	if len(ip.Status.IPMappings) == 0 {
```
Comment (aleoli) [inline pkg/liqo-controller-manager/networking/external-network/remapping/ip_controller.go:94]: Why that?

**Review by aleoli — APPROVED** [review]

File: pkg/liqo-controller-manager/networking/external-network/remapping/ip_controller.go (hunk 1 above)
Comment (cheina97) [reply pkg/liqo-controller-manager/networking/external-network/remapping/ip_controller.go:94]: Because it is not necessary, len(nil) return 0

**Pull request merged**